cupy/cuda/stream.pyx

```diff
@@ -291,23 +301,33 @@ class Stream(BaseStream):
 
     Args:
         null (bool): If ``True``, the stream is a null stream (i.e. the default
-            stream that synchronizes with all streams). Otherwise, a plain new
-            stream is created. Note that you can also use ``Stream.null``
-            singleton object instead of creating new null stream object.
-        non_blocking (bool): If ``True``, the stream does not synchronize with
-            the NULL stream.
+            stream that synchronizes with all streams). Note that you can also
+            use ``Stream.null`` singleton object instead of creating new null
+            stream object.
+        ptds (bool): If ``True`` and ``null`` is ``False``, the per-thread
+            default stream is used. Note that you can also use ``Stream.ptds``
+            singleton object instead of creating new per-thread default stream
+            object.
```

Comment:
I'm not entirely sure this is the right place, perhaps it would fit best in https://github.com/cupy/cupy/blob/master/docs/source/reference/environment.rst ?

Reply:
I've added documentation for the environment variable in https://github.com/cupy/cupy/pull/4322/commits/7acb9c4e48ccb3cc624f04d4eadd780c6a04d0fe